styles.css

```diff
@@ -1 +1,79 @@
-/*Maak hier jouw CSS!*/
```

Comment:
Waarom heb je hier allerlei marges op gezet? Om de pagina van de randen af te zetten? Ik denk niet dat het fout is, maar ben gewoon zelf benieuwd. Ik had dit namelijk niet gedaan.

Reply:
Ja inderdaad overbodig achteraf gezien... thanks voor je feedback!